source/text/labelrenderpass.ts

```diff
@@ -260,34 +244,38 @@ export class LabelRenderPass extends Initializable {
         gl.uniform2fv(this._uNdcOffset, this._ndcOffset);
         gl.uniformMatrix4fv(this._uViewProjection, gl.GL_FALSE, this._camera.viewProjection);
 
-        this._font.glyphTexture.bind(gl.TEXTURE0);
-
-
         /* Controlling renderer is expected to bind the appropriate target, thus, unbinding is not
         necessary. */
         this._target.bind();
 
-        this._geometry3D.bind();
-        this._geometry3D.draw();
+        this._geometry.bind();
 
-        gl.uniformMatrix4fv(this._uViewProjection, gl.GL_FALSE, mat4.create());
+        for (let i = 0; i < this._labels.length; ++i) {
+            const label = this._labels[i];
+            const range = this._ranges[i];
+            if (range[0] === range[1]) {
+                continue;
```

Comment:
so, it is an empty label? (just for my comprehension)

Reply:
yes